decidim-accountability/app/views/decidim/accountability/results/_project.html.erb

```diff
@@ -10,9 +10,6 @@
     <%= cell "decidim/tags", result %>
   </div>
 
-  <%# REDESIGN_PENDING: Both comments and attachments sections are pending %>
-  <%= attachments_for result %>
```

Comment:
are we removing all the attachments on a project?

Reply:
No, the attachments are included in the project cell, which combines all the tabs at the same point